packages/x-components/src/x-modules/facets/components/facets/facets.vue

```diff
@@ -39,6 +39,7 @@
   import { Component, Prop, Vue } from 'vue-property-decorator';
   import { Getter } from '../../../../components/decorators/store.decorators';
   import { xComponentMixin } from '../../../../components/x-component.mixin';
+  import { toKebabCase } from '../../../../utils';
```

Comment:
Import directly from the file of the function, let's try to avoid barrels inside our code please.